src/sass/claims-status.scss

```diff
@@ -218,128 +208,118 @@ a.claim-list-item {
   }
 }
 
-.claims-status {
-  i.fa {
-    font-size: 24px;
-  }
+.claims-status-icon {
+  font-size: 24px;
+}
 
-  .next-action-container {
-    padding-right: 1em;
-  }
+.next-action-container {
+  padding-right: 1em;
+}
 
-  .next-action {
-    padding: 1em;
-    position: relative;
+.next-action {
+  padding: 1em;
+  position: relative;
 
-    p {
-      margin-top: 0;
-    }
+  &.hearing {
+    background: $color-inset-bg;
+  }
 
-    &.hearing {
-      background: $color-inset-bg;
-    }
+  &.soc, &.ssoc {
+    background: $color-gibill-accent;
+    padding-left: 2.5em;
 
-    &.soc, &.ssoc {
-      background: $color-gibill-accent;
-      padding-left: 2.5em;
-
-      &:before {
-        left: 1em;
-        top: 1.5em;
-        position: absolute;
-        font-size: 1.25em;
-        display: inline-block;
-        font: normal normal normal 14px/1 FontAwesome;
-        font-size: inherit;
-        text-rendering: auto;
-        -webkit-font-smoothing: antialiased;
-        -moz-osx-font-smoothing: grayscale;
-        content: "\F071";
-      }
+    &:before {
+      left: 1em;
+      top: 1.5em;
+      position: absolute;
+      font-size: 1.25em;
+      display: inline-block;
+      font: normal normal normal 14px/1 FontAwesome;
+      font-size: inherit;
+      text-rendering: auto;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+      content: "\F071";
     }
   }
+}
 
-  .last-status {
-    margin-top: 1em;
-    padding: 0 1em 1em 0;
+.claims-first-paragraph {
+  margin-top: 0;
+}
 
-    h4:first-of-type {
-      padding: 0;
-    }
+.last-status {
+  margin-top: 1em;
+  padding: 0 1em 1em 0;
+}
 
-    .content {
-      position: relative;
-      padding-left: 3.25rem;
+.last-status-text {
+  h5 + & {
+    margin-top: 0;
+  }
+}
```

Comment:
Thanks @jbalboni, unless the mocks indicate that there should be a margin between h5 and p (such as in the `bva_final_decision` case in `appealStatusDescriptions`), how about we play it safe and simply add both cases where `margin-top` should be zero (all previous activity `p`, and when there is an instance of `h5` + `p` as described above in last status) as rules to `claims-first-paragraph`, so:
```
.claims-first-paragraph {
  .h4 + &, h5 + & {
    margin-top: 0;
  }
}
```